response.go

```diff
@@ -0,0 +1,97 @@
+package webfmwk
+
+import (
+	"fmt"
+	"unicode/utf8"
+)
+
+type (
+	Header [2]string
+
+	SendResponse interface {
+		JSONResponse
+		XMLResponse
+
+		// SendResponse create & send a response according to the parameters
+		SendResponse(op int, content []byte, headers ...Header) error
+
+		// SetHeader set the header of the http response
+		SetHeaders(headers ...Header)
+
+		SetHeader(k, v string)
+
+		// IsPretty toggle the compact output mode
+		IsPretty() bool
+	}
+
+	XMLResponse interface {
+		// JSONBlob answer the JSON content with the status code op
+		XMLBlob(op int, content []byte) error
+	}
+)
+
+// SetHeaders implement Context
+func (c *icontext) SetHeaders(headers ...Header) {
+	c.setHeaders(headers...)
+}
+
+// SetHeaders implement Context
+func (c *icontext) SetHeader(k, v string) {
+	c.setHeaders(Header{k, v})
+}
+
+// IsPretty implement Context
+func (c *icontext) IsPretty() bool {
+	return len(c.query[_prettyTag]) > 0
+}
+
+// XMLBlob sent a XML response already encoded
+func (c *icontext) XMLBlob(statusCode int, content []byte) error {
+	c.setHeaders(Header{"Content-Type", "application/xml; charset=UTF-8"},
+		Header{"Produce", "application/xml; charset=UTF-8"})
+	return c.response(statusCode, content)
+}
+
+// SendResponse implement Context
+func (c *icontext) SendResponse(statusCode int, content []byte, headers ...Header) error {
+	c.setHeaders(headers...)
+	return c.response(statusCode, content)
+}
+
+// setHeader set the header of the holded http.ResponseWriter
+func (c *icontext) setHeaders(headers ...Header) {
+	for _, h := range headers {
+		key, val := h[0], h[1]
+		if key == "" || val == "" {
+			c.log.Warnf("can't set header [%s] to [%s] (empty value)", key, val)
+
+			return
+		}
+
+		c.w.Header().Set(key, val)
+	}
+}
+
+// response generate the http.Response with the holded http.ResponseWriter
+// IDEA: add toggler `logReponse` ?
+func (c *icontext) response(statusCode int, content []byte) error {
+	var l = len(content)
+
+	c.log.Infof("[%d](%d)", statusCode, l)
```

Comment:
let's dump, and introduce in the handler/logger an out response w/ timer
 
imho outputing content of the response should be togglable